bin/list.sh

```diff
@@ -1,34 +1,38 @@
 #!/bin/bash -f
 #
 # This script lists databases, terminologies, versions, graphs
-# available in the configured stardog and elasticsearch.
+# available in the configured graph db and elasticsearch.
 # The --noconfig flag is for running in the dev environment
 # where the setenv.sh file does not exist.
 #
 config=1
 ncflag=""
 help=0
 quiet=0
-stardog=1
+graph_db=1
 es=1
+databases=("NCIT2" "CTRP")
+
+DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+
 while [[ "$#" -gt 0 ]]; do case $1 in
     --help) help=1;;
     # use environment variable config (dev env)
     --noconfig) config=0; ncflag="--noconfig";;
     # avoid printing header/footer
     --quiet) quiet=1;;
-    # show stardog data only
-    --stardog) es=0;;
+    # show graph DB data only
+    --graph_db) es=0;;
     # show es data only
```

Comment:
Here you're calling it graph_db but in the --help it's "graphdb".  I think just to avoid underscores, if we make the switch --graphdb that's key, though ideally internally we'd call the variable graphdb also and not graph_db.